0001-kaa-protocol/README.md

```diff
@@ -123,9 +123,9 @@ For example, use `/json` for JSON-formatted payload, and `/protobuf/<scheme_id>`
 
 ### Request/response pattern
 Many extensions require request/response style communication, which is not supported by MQTT natively.
-That is overcome by introducing a separate topic name for responses.
+That is overcome by introducing a separate resource path for responses.
```

Comment:
Having given this another thought, I suggest making it more explicit that this design is for MQTT only. It's not explicit enough right now.